modules/vpc-network/variables.tf

```diff
@@ -65,3 +65,8 @@ variable "enable_flow_logging" {
   default     = true
 }
 
+variable allowed_public_restricted {
```

Comment:
Maybe make it clearer in the variable name / documentation that these are CIDR blocks.

Reply:
Good point. Updated with a suggestion to name it `allowed_public_restricted_subnetworks`.